components/atlas-uvs/examples/static/index-old.html

```diff
@@ -0,0 +1,49 @@
+<html>
+    <head>
+        <script src="https://aframe.io/releases/1.1.0/aframe.min.js"></script>
```

Comment:
Do we need an `index-old` file?

Reply:
It was intended to display the earlier version of the component working with A-Frame 1.1 and earlier as expected.

Reply:
The example is exactly the same except for `<script src="https://unpkg.com/aframe-atlas-uvs-component@2.1.0/dist/aframe-atlas-uvs-component.min.js"></script>` right? Maybe it's not necessary. We already indicate in the readme to use `2.1.0` for older A-Frame versions

Reply:
Yes you're right, so I can just remove index-old as it's hopefully not used very much and can just be explained in the readme to use 2.1.0

Reply:
removed